cmd/url.go

```diff
@@ -30,3 +33,39 @@ var urlDecodeCmd = &cobra.Command{
 		fmt.Println(s)
 	},
 }
+
+var urlToJSON = &cobra.Command{
+	Use:                   "url-to-json [text]",
+	Short:                 "Convert url to JSON",
+	Args:                  cobra.ExactArgs(1),
+	DisableFlagsInUseLine: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		s, err := url.Parse(args[0])
+		if err != nil {
+			fmt.Println("Invalid query")
+		}
+
+		type result struct {
+			Scheme string
+			Host   string
+			Path   string
+			Query  map[string]string
+		}
+		res := &result{
+			Scheme: s.Scheme,
+			Host:   s.Host,
+			Path:   s.Path,
+		}
+
+		if s.RawQuery != "" {
+			res.Query = make(map[string]string)
+			for _, v := range strings.Split(s.RawQuery, "&") {
+				param := strings.Split(v, "=")
+				res.Query[param[0]] = param[1]
+			}
```

Comment:
You can use `Query()` method to do this

Reply:
Thanks for your advice!

I have been used.